ui/job-view/headerbars/WatchedRepo.jsx

```diff
@@ -137,9 +137,15 @@ class WatchedRepo extends React.Component {
     return (
       <ButtonGroup>
         <Button
-          onClick={() => {
+          href={updateRepoParams(watchedRepo)}
+          onClick={(event) => {
             pushRoute({ search: updateRepoParams(watchedRepo) });
             this.updateTreeStatus(true);
+            event.preventDefault();
+          }}
+          style={{
+            // NOTE: This is a hack to make this align with drop down button.
+            padding: '.375rem .75rem',
```

Comment:
Nit: We try to limit inline styles except where its unavoidable such as in the graphing library we use.